material.inc.php

```diff
@@ -63,15 +63,15 @@
   1 => array(
     array('name'=>'blueprints', 'choice_description' => self::_('any tile to peek')),
     array('name'=>'crowbar', 'choice_description' => self::_('an adjacent tile to disable')),
-    array('name'=>'crystal-ball'),
+    array('name'=>'crystal-ball', 'choice_description' => self::_('to reorder the 3 upcoming events')),
     array('name'=>'donuts', 'choice_description' => self::_('any guard to lose all movement for one turn')),
     array('name'=>'dynamite', 'choice_description' => self::_('an adjacent wall to remove')),
     array('name'=>'emp'),
     array('name'=>'invisible-suit'),
     array('name'=>'makeup-kit'),
     array('name'=>'rollerskates'),
     array('name'=>'smoke-bomb'),
-    array('name'=>'stethoscope'),
+    array('name'=>'stethoscope', 'choice_description' => self::_('if you want to reroll one die using the Stethoscope')),
```

Comment:
They don't reroll, they change it to a side of their choice

Reply:
Good news is this is what I implemented ;)
I changed the choice_description